src/main/resources/liquibase/crop_changelog/22_3_0.xml

```diff
@@ -62,4 +62,40 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="cuenyad" id="v22.3.0-5">
+		<preConditions onFail="MARK_RAN">
+				<sqlCheck expectedResult="1">
+					SELECT count(1) > 0 FROM CVTERMPROP WHERE CVTERM_ID = 8201 AND value = 'Germplasm Descriptor';
+				</sqlCheck>
+		</preConditions>
+		<comment>Recategorize OBS_UNIT_ID as Experimental Design variable</comment>
+		<sql dbms="mysql" splitStatements="true">
+			set @OBS_UNIT_ID = (select cvterm_id from cvterm where name ='OBS_UNIT_ID' and cv_id = 1040);
+
+			# update the type of variable as Germplasm Descriptor.
+								  UPDATE CVTERMPROP SET value = 'Experimental Design' WHERE CVTERM_ID = @OBS_UNIT_ID;
+
+			# update the Germplasm Descriptor cvterm_id by Experimental Design in the projectprop table.
+				  UPDATE projectprop SET type_id = 1810  WHERE variable_id = @OBS_UNIT_ID;
```

Comment:
@cuenyad for safety I think we should add the type_id condition in the where clause